src/ucp/core/ucp_ep.c

```diff
@@ -1719,3 +1870,29 @@ size_t ucp_ep_config_get_zcopy_auto_thresh(size_t iovcnt,
 
     return zcopy_thresh;
 }
+
+ucs_status_t ucp_ep_add_connected_lane(ucp_ep_h ucp_ep, uct_ep_h uct_ep)
```

Comment:
the configuration key should be created up-front with a "connected lane"
since at ep creation time we already know it's sockaddr

Reply:
but we don't know UCT iface